api/scpca_portal/utils/helpers.py

```diff
@@ -42,6 +42,14 @@ def join_workflow_versions(workflow_versions: Set) -> str:
     return ", ".join(sorted(set(workflow_versions)))
 
 
+def get_chunk_list(list: List[Any], size: int) -> Generator[List[Any], None, None]:
+    """
+    Yields chunks of the given list with the specified size.
+    """
+    for i in range(0, len(list), size):
+        yield list[i : i + size]
+
+
```

Comment:
I've used the solution from [Stack overflow: Split a list into equally-sized chunks](https://stackoverflow.com/a/312464) for list chunking, as using generators seems to be a commonly used approach. Please let me know if you have any other suggestions!